### PR TITLE
Set up ECS deployment workflow

### PR DESCRIPTION
**Pull request by ianwestcott**
Asana Task: [Create Deployment pipeline](https://app.asana.com/0/1200032851949029/1200298390819078/f)

### CODE REVIEW

**Comment by ianwestcott** [comment]
Hm, is there a way to test the workflow before merging it?

**Comment by paulswartz** [comment]
@ianwestcott 
> Hm, is there a way to test the workflow before merging it?

Kind of. You can fork the repository and configure your fork with the same credentials. Then you can iterate to your heart's content before submitting the PR.

**Comment by ianwestcott** [comment]
@manny-m I pulled in your changes from the [`mm-docker-config`](https://github.com/mbta/parking_lpr/tree/mm-docker-config) branch, and made some tweaks to the Dockerfile to support building JavaScript assets as a separate build stage using the official `nodejs` container instead of having to manually install Node.js via `curl | bash`. I confirmed that the built container runs, though I'm seeing some errors around the configuration that may need to be dealt with in a follow-up change. Hopefully it'll be easier to tell what configuration is needed once we have this deployed to ECS.

**Comment by ianwestcott** [comment]
As for the GitHub Actions workflow, I was able to deploy successfully in my personal fork: https://github.com/ianwestcott/parking_lpr/runs/2585290064

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  libssl1.1 libsctp1 curl \
+  && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /root
+EXPOSE 4000
+ENV MIX_ENV=prod TERM=xterm LANG="C.UTF-8" PORT=4000
+
+COPY --from=app-builder /root/_build/prod/rel/parking_lpr .
```
Comment (arkadyan) [inline Dockerfile:48]: Some of our apps also include a line to copy in static assets? (e.g. [screens](https://github.com/mbta/screens/blob/master/Dockerfile#L50)) Are you getting assets properly without that?

**Review by arkadyan — APPROVED** [review]
Included a small question about copying static assets, but otherwise looks good to me if it's working. 👍

File: Dockerfile (hunk 1 above)
Comment (ianwestcott) [reply Dockerfile:48]: I don't know the answer to this; @manny-m can you investigate?

**Pull request closed without merging**